napari/_vispy/vendored/volume.py

```diff
@@ -375,11 +475,42 @@
         """,
 )
 
+# This is an iso shader for categorical data (e.g., label images)
+ISO_CATEGORICAL_SNIPPETS = dict(
+    before_loop="""
+        vec4 color3 = vec4(0.0);  // final color
+        vec3 dstep = 1.5 / u_shape;  // step to sample derivative
```

Comment:
1.5 was chosen to be sure you end up inside the label?

Reply:
Yeah, this is arbitrary, but it matches the value from the "normal" iso shader.

Reply:
^ I've added a comment to indicate that this is the case.